src/WebformBuilder.js

```diff
@@ -334,7 +334,17 @@ export default class WebformBuilder extends Component {
         default: index === 0,
       };
 
-      eachComponent(resource.components, (component) => {
+      let { components } = resource;
+      if (!Array.isArray(components)) {
+        try {
+          components = JSON.parse(components);
```

Comment:
In what case does this happen?  This seems like a band-aid over a more fundamental problem. The components should never get passed in as a JSON string.

Reply:
This was hotfix for that issue: https://github.com/formio/formio.js/issues/1625
I've asked Randall to provide more info about use case, but he advised me to only check if components is an array. So maybe we need to ask them to provide their environment, because I can't reproduce it on my machine.

Reply:
I think we need to investigate why this is coming in as a string. I don't want to include a hack to handle a malformed form.

Reply:
I don't think we will pull this in. We need to figure out why the "components" is being sent in as a string and not an array. That is the real bug we should be fixing instead of this... People may start abusing this new "feature"